src/main/webapp/app/shared/pipes/safe-resource-url.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Pipe({ name: 'safeResourceUrl' })
+export class SafeResourceUrlPipe implements PipeTransform {
```

Comment:
Would also be nice to have some documentation here

Reply:
I added the documentation